src/core_plugins/kibana/public/management/app.html

```diff
@@ -14,8 +14,8 @@
 
       <!-- Tabs. -->
       <div data-transclude-slot="bottomRow" class="kuiLocalTabs">
-        <header ng-if="!sectionName" id="tabHeader" tabindex="0">
-          <h2 class="kuiLocalTab">
+        <header ng-if="!sectionName" id="tabHeader">
+          <h2>
```

Comment:
This tab should have the class `kuiLocalTab-isSelected` applied to it, which should do the trick.